src/blob.rs

```diff
@@ -298,9 +298,6 @@ impl<'a> BlobObject<'a> {
     }
 
     /// Returns the extension of the blob.
-    ///
-    /// If a blob's filename has an extension, it is always guaranteed
-    /// to be lowercase.
     pub fn suffix(&self) -> Option<&str> {
```

Comment:
there is at least one place i know of that relies on the old definition of the suffix to be lowercase: there is a comparison against the webxdc extension